playkit/src/main/java/com/kaltura/playkit/profiler/ExoPlayerProfilingListener.java

```diff
@@ -147,17 +147,9 @@ public void onPositionDiscontinuity(EventTime eventTime, int reason) {
         String reasonString;
         switch (reason) {
             case DISCONTINUITY_REASON_PERIOD_TRANSITION:
-                reasonString = "PeriodTransition";
```

Comment:
@GouravSna 
I think here we have to update the reason string and not to merge the switch

Reply:
Done